<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelGameFolder">
          <property name="text">
           <string>Game Folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditGameFolder"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonGameFolder">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelPackManagerFolder">
          <property name="text">
           <string>Pack Manager Folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPackManagerFolder"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonPackManagerFolder">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="labelSaveFolder">
          <property name="text">
           <string>Save Folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSaveFolder"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSaveFolder">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelTmpFolder">
          <property name="text">
           <string>Tmp Folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditTmpFolder"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonTmpFolder">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxInstallMod">
        <property name="text">
         <string>Install Mod</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxRemoveTmpFolder">
        <property name="text">
         <string>Remove Tmp Folder</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
